### PR TITLE
fix(cli): serverless ctx mock pkg move bug

### PR DESCRIPTION
**Pull request by woorui**
# Description

1. Move `MockContext` back to `serverless/mock` package.
2. Change function signature from `ReadLLMFunctionCall(fnCall any) error` to `ReadLLMFunctionCall() (*ai.FunctionCall, error)`, This can avoid type missing.

### CODE REVIEW

File: serverless/guest/context_llm.go [hunk 1]
```diff
@@ -8,6 +10,6 @@ func (c *GuestContext) WriteLLMResult(result string) error {
 	panic("not implemented")
 }
 
-func (c *GuestContext) ReadLLMFunctionCall(fnCall any) error {
+func (c *GuestContext) ReadLLMFunctionCall() (*ai.FunctionCall, error) {
```
Comment (venjiang) [inline serverless/guest/context_llm.go:13]: Is there a better name?

Comment (woorui) [reply serverless/guest/context_llm.go:13]: Do you have any suggestion, I think this variable name is not bad.

Comment (venjiang) [reply serverless/guest/context_llm.go:13]: I suggest remove `Read`, like this: `LLMFunctionCall()`

**Review by fanweixiao — APPROVED** [review]

**Comment by fanweixiao** [comment]
great job

**Pull request merged**